web/themes/custom/contribtracker/templates/block/block--contribgraph.html.twig

```diff
@@ -0,0 +1,37 @@
+{% block contributed_graph_content %}
```

Comment:
Rather than using a block type, we should create our own block (not based on block content type).